exercises/random_number.cpp

```diff
@@ -4,3 +4,16 @@
   Output:
   The random number is: 4
 */
+#include <cstdlib>
+#include <iostream>
+
+#define EXIT_SUCCESS 0
+
+using namespace std;
+
+int main() {
+
+  cout << "The random number id: " << rand() % 300;
```

Comment:
forse intendevi "is"?
```suggestion
  cout << "The random number is: " << rand() % 300;
```